typescript/sdk/src/token/AltVMWarpModule.ts

```diff
@@ -391,38 +382,30 @@ export class AltVMWarpModule<PT extends ProtocolType> extends HyperlaneModule<
    *
    * @param chain - The chain to deploy the module on.
    * @param config - The configuration for the token router.
-   * @param multiProvider - The multi-provider instance to use.
+   * @param chainLookup - Chain metadata lookup functions
    * @param signer - The AltVM signing client
    * @returns A new instance of the AltVMWarpModule.
    */
-  static async create<PT extends ProtocolType>(params: {
-    chain: ChainNameOrId;
+  static async create(params: {
+    chain: string;
```

Comment:
i suppose this is part of detangling the types

Reply:
i think we should actually just use chainName only in places instead of conflating name/id, but again something not for this PR